Directory.Build.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <PropertyGroup>
```

Comment:
This looks like the previous PR that was made to update the version - https://github.com/microsoft/ntosebpfext/pull/111/files

Do we need to update this?

Reply:
added ebpf_ext_version.h update to the script